inc/libum.h

```diff
@@ -604,6 +602,20 @@ LIBUM_SHARED_EXPORT int um_take_step(um_state *hndl, const int dev,
 
 LIBUM_SHARED_EXPORT int um_cmd_options(um_state *hndl, const int optionbits);
 
+/**
+ * @brief Send a command to a device.
+ *
+ * @param   hndl    Pointer to session handle
+ * @param   dev     Device ID
+ * @param   cmd     Command ID
+ * @param   argc    Number of arguments (may be zero)
+ * @param   argv    Pointer to command argument array (may be NULL)
+ * @return  Negative value if an error occured. Zero or positive value otherwise
+ */
+
+LIBUM_SHARED_EXPORT int um_cmd(um_state *hndl, const int dev, const int cmd,
```

Comment:
I don't see any documentation for what possible command ids exist. I'm guessing this function is not really meant for users? If so, please document it as such. Without that, I'm worried someone might accidentally send dangerous commands.